meson.build

```diff
@@ -276,4 +276,9 @@ if get_option('apps')
 endif
 
 pkg = import('pkgconfig')
-pkg.generate(nanoarrow_lib)
+pkg.generate(libraries : nanoarrow_lib,
+             subdirs : [incdir],
```

Comment:
I am not an expert in pkgconfig but are you sure this is correct? I don't think there will be a src directory with the installation